providers/web3.tsx

```diff
@@ -9,6 +9,12 @@ import { getStaticRpcBatchProvider } from '@lido-sdk/providers';
 
 import { useClientConfig } from 'providers/client-config';
 import { dynamics, useGetRpcUrlByChainId } from 'config';
+import { OrbyProvider } from '@orbykit/react';
+
+const orbyConfig = {
+  clientId: 'orblabs-internal-frontend',
+  clientSecret: 'orblabs-internal-frontend',
+};
```

Comment:
Let's pass these via environment variables

Reply:
Added - be sure to add the below to your `.env.local`

```
ORBY_CLIENT_ID="orblabs-internal-frontend"
ORBY_CLIENT_SECRET="orblabs-internal-frontend"
```